From 61ded4dae1bc9bfd3c8d0615c0ef9f4e3897ee18 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Mon, 30 Oct 2017 21:05:46 +0100
Subject: [PATCH 07/10] adb: Allow adbd to be ran as root

---
 adb/daemon/main.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/adb/daemon/main.cpp b/adb/daemon/main.cpp
index 4721e2fbb4..adcb1a4111 100644
--- a/adb/daemon/main.cpp
+++ b/adb/daemon/main.cpp
@@ -67,6 +67,7 @@ static bool should_drop_privileges() {
 #if defined(ALLOW_ADBD_ROOT)
     char value[PROPERTY_VALUE_MAX];
 
+    return false;
     // The properties that affect `adb root` and `adb unroot` are ro.secure and
     // ro.debuggable. In this context the names don't make the expected behavior
     // particularly obvious.
-- 
2.17.0

